packages/synthetic-chain/public/upgrade-test-scripts/run_test.sh

```diff
@@ -24,13 +24,23 @@ echo '
  |_____| |_____|    |    |_____  |_____| |  |  | |______
 '
 
+source ./env_setup.sh
+
+cd /usr/src/proposals/"$PROPOSAL/" || fail "Proposal $PROPOSAL does not exist"
+
+if test -f prepare-test.sh
+then
+  echo "[$PROPOSAL] Running prepare-test.sh"
+  ./prepare-test.sh
```

Comment:
@usmanmani1122 
Will we create this script in a separate PR?
Also what exactly we gonna do in it?

Reply:
I think with @turadg we were thinking of adding a hook in the [docker build phase](https://github.com/Agoric/agoric-3-proposals/blob/ce1df1dfef039d1cf073512730fe4e7ad1b41125/packages/synthetic-chain/src/cli/dockerfileGen.ts#L166-L172) because I thought we needed something in the image itself, but now I'm wondering if we in fact need that, or if we can't just run the hook before starting agd as you're doing here.